tgui/src/interfaces/mech_bay_power_console.ract

```diff
@@ -0,0 +1,42 @@
+<script>
+  component.exports = {
+    data: {
+      mechChargeState(charge) {
+		if(charge >= (this.get('adata.recharge_port.mech.cell.maxcharge')/2)) return 'good'
+		else if(charge >= (this.get('adata.recharge_port.mech.cell.maxcharge')/4)) return 'average'
+		else return 'bad'
+        },
+	  mechHealthState(health) {
+	    if(health > (this.get('adata.recharge_port.mech.maxhealth')/2)) return 'good'
+		else if(health > (this.get('adata.recharge_port.mech.maxhealth')/4)) return 'average'
+		else return 'bad'
+	    }
+      }
+    }
+</script>
+
+<ui-display title='Mech Status'>
+  {{#if data.recharge_port}}
+	{{#if data.recharge_port.mech}}
+	  <ui-section label='Integrity'>
+	    <ui-bar min='0' max='{{adata.recharge_port.mech.maxhealth}}' value='{{adata.recharge_port.mech.health}}'state='{{mechHealthState(adata.recharge_port.mech.health)}}'>{{Math.round(adata.recharge_port.mech.health)}}/{{adata.recharge_port.mech.maxhealth}}</ui-bar>	
+      </ui-section>
+	  {{#if data.recharge_port.mech.cell}}
+		{{#if data.recharge_port.mech.cell.critfail}}
+		  <ui-section label='Power'>WARNING the mech cell seems faulty!</ui-section>
+		{{else}}
+          <ui-section label='Power'>
+            <ui-bar min='0' max='{{adata.recharge_port.mech.cell.maxcharge}}' value='{{adata.recharge_port.mech.cell.charge}}'state='{{mechChargeState(adata.recharge_port.mech.cell.charge)}}'>{{Math.round(adata.recharge_port.mech.cell.charge)}}/{{Math.round(adata.recharge_port.mech.cell.maxcharge)}}</ui-bar>		  
+		  </ui-section>
+		{{/if}}
+	  {{else}}
+		<ui-section label='Power'>WARNING the mech cell is missing!</ui-section>
```

Comment:
the WARNING came from the original UI actually, but it had a colon on it, I dropped that since the "Power" label already has a colon.